### PR TITLE
Add QuickSwap

### PR DESCRIPTION
**Pull request by pollastri-pierre**
This PR adds QuickSwap to the erc7730 registry

### CODE REVIEW

File: registry/quickswap/calldata-QuickSwap.json [hunk 1]
```diff
@@ -0,0 +1,555 @@
...
+            "0x38ed1739": {
+                "$id": "swapExactTokensForTokens",
+                "intent": "Swap",
+                "fields": [
+                    {
+                        "path": "params.amountIn",
+                        "label": "Amount to Send",
+                        "format": "tokenAmount",
+                        "params": {
+                            "tokenPath": "params.path[0]"
```
Comment (jnicoulaud-ledger) [inline registry/quickswap/calldata-QuickSwap.json]:
```suggestion
                            "tokenPath": "params.path.[0]"
```

File: registry/quickswap/calldata-QuickSwap.json [hunk 2]
```diff
@@ -0,0 +1,555 @@
...
+            "0x18cbafe5": {
+                "$id": "swapExactTokensForETH",
+                "intent": "Swap",
+                "fields": [
+                    {
+                        "path": "params.amountIn",
+                        "label": "Amount to Send",
+                        "format": "tokenAmount",
+                        "params": {
+                            "tokenPath": "params.path[0]"
```
Comment (jnicoulaud-ledger) [inline registry/quickswap/calldata-QuickSwap.json]:
```suggestion
                            "tokenPath": "params.path.[0]"
```

Comment (jnicoulaud-ledger) [reply registry/quickswap/calldata-QuickSwap.json]: There are more, a search and replace will be more efficient ^^

File: registry/quickswap/calldata-QuickSwap.json [hunk 3]
```diff
@@ -0,0 +1,555 @@
...
+                        "path": "params.to",
+                        "label": "Beneficiary",
+                        "format": "addressName"
+                    },
+                    {
+                        "path": "params.deadline",
+                        "label": "Deadline",
+                        "format": "date",
+                        "params": {
+                            "encoding": "blockheight"
```
Comment (lcastillo-ledger) [inline registry/quickswap/calldata-QuickSwap.json]: Quickswap looks like a duplicate of Uniswap Router02, and in Router02 addLiquidity looks like its encoded using a uni timestamp (https://docs.uniswap.org/contracts/v2/reference/smart-contracts/router-02#addliquidity). The encoding here should probably be "timestamp"

File: registry/quickswap/calldata-QuickSwap.json [hunk 4]
```diff
@@ -0,0 +1,555 @@
...
```
Comment (lcastillo-ledger) [inline registry/quickswap/calldata-QuickSwap.json]: Same, probably should be a timestamp

File: registry/quickswap/calldata-QuickSwap.json [hunk 5]
```diff
@@ -0,0 +1,555 @@
...
```
Comment (lcastillo-ledger) [inline registry/quickswap/calldata-QuickSwap.json]: timestamp

File: registry/quickswap/calldata-QuickSwap.json [hunk 6]
```diff
@@ -0,0 +1,555 @@
...
```
Comment (lcastillo-ledger) [inline registry/quickswap/calldata-QuickSwap.json]: timestamp

File: registry/quickswap/calldata-QuickSwap.json [hunk 7]
```diff
@@ -0,0 +1,555 @@
...
```
Comment (lcastillo-ledger) [inline registry/quickswap/calldata-QuickSwap.json]: timestamp

File: registry/quickswap/calldata-QuickSwap.json [hunk 8]
```diff
@@ -0,0 +1,555 @@
...
```
Comment (lcastillo-ledger) [inline registry/quickswap/calldata-QuickSwap.json]: timestamp

File: registry/quickswap/calldata-QuickSwap.json [hunk 9]
```diff
@@ -0,0 +1,555 @@
...
```
Comment (lcastillo-ledger) [inline registry/quickswap/calldata-QuickSwap.json]: timestamp

File: registry/quickswap/calldata-QuickSwap.json [hunk 10]
```diff
@@ -0,0 +1,555 @@
...
```
Comment (lcastillo-ledger) [inline registry/quickswap/calldata-QuickSwap.json]: timestamp

**Review by jnicoulaud-ledger — COMMENTED** [review]
@pollastri-pierre CI is running now, you just need to remove all the `params.` prefixes I think

**Comment by pollastri-pierre** [comment]
Thanks @jnicoulaud-ledger all green now 🙂

**Review by jnicoulaud-ledger — APPROVED** [review]

**Pull request merged**